wiki/useful-windows-tips.md

```diff
@@ -0,0 +1,42 @@
+# Useful tips and "hacks" for Windows
+
+### Get more options when right-clicking
+
+When right-clicking on a file or folder, you can hold down shift to get more options, such as to "Copy as path" which lets you copy the path of the file.
+
+
+### Windows Master Control Panel shortcut
+
+The Windows Master Control Panel shortcut, often referred to as "God Mode", is a shortcut to access all control panel settings in one place.
+
+To create the shortcut, create a folder and name it
+```
+allSettings.{ED7BA470-8E54-465E-825C-99712043E01C}
+```
+The text in front of the dot has no effect.
+
+You can now open the folder to find all control panel settings.
+
+### Make Windows search faster
+
+You can make the Windows Search way faster, by enabling Indexing, which indexes the locations you select. In simple words, it essentially creates a list of all files, so that when you search for a file, Windows searches in that list instead.
+
+First, search for "Indexing Options" in the Start menu. Then, in the window that appears, click "Modify". Then, click "Show All Locations". Now, check all the locations that you want Windows to index. Finally, click Ok to confirm. Your files will now be indexed.
+
+Keep in mind, indexing can take a long time depending on how many files it has to go through, and can take up a few resources. This will only happen once though.
```

Comment:
Will change that and i'll add a note that it does indeed try to not use up too many resources (which it does do).

Reply:
@FloatingMilkshake Changed it to :

"Keep in mind, indexing can take a long time depending on how many files it has to go through, and it may consume some system resources while indexing is in-progress. Indexing will pause however, if it detects that you are using your computer. The indexing process only needs to run once though."